server/database/init.sql

```diff
@@ -12,7 +12,7 @@ CREATE TABLE devices (
     owner       INTEGER NOT NULL REFERENCES accounts(id) ON DELETE CASCADE,
     device_id   TEXT NOT NULL,
     name        BYTEA NOT NULL,
-    auth_token  VARCHAR(255) NOT NULL UNIQUE,
+    auth_token  BYTEA NOT NULL,
```

Comment:
This should also, if not already, be changed in the quries

Reply:
Already was changed